contracts/pcv/utils/PCVDepositWrapper.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.4;
+
+import "../IPCVDeposit.sol";
+
+/**
+  @notice a lightweight contract to wrap old PCV deposits to use the new interface 
+  @author Fei Protocol
+  When upgrading the PCVDeposit interface, there are many old contracts which do not support it.
+  The main use case for the new interface is to add read methods for the Collateralization Oracle.
+  Most PCVDeposits resistant balance method is simply returning the balance as a pass-through
+  If the PCVDeposit holds FEI it may be considered as protocol FEI
+
+  This wrapper can be used in the CR oracle which reduces the number of contract upgrades and reduces the complexity and risk of the upgrade
+*/
+contract PCVDepositWrapper {
+   
+    /// @notice the referenced PCV Deposit
+    IPCVDeposit public pcvDeposit;
+
+    /// @notice the balance reported in token
+    address public token;
+
+    /// @notice a flag for whether to report the balance as protocol owned FEI
+    bool public isProtocolFeiDeposit;
+
+    constructor(IPCVDeposit _pcvDeposit, address _token, bool _isProtocolFeiDeposit) {
+        pcvDeposit = _pcvDeposit;
+        token = _token;
+        isProtocolFeiDeposit = _isProtocolFeiDeposit;
+    }
+
+    /// @notice returns total balance of PCV in the Deposit
+    function balance() public view returns (uint256) {
+        return pcvDeposit.balance();
+    }
+
+    /// @notice returns the resistant balance and FEI in the deposit
+    function resistantBalanceAndFei() public view returns (uint256, uint256) {
+        uint256 resistantBalance = balance();
+        uint256 reistantFei = isProtocolFeiDeposit ? resistantBalance : 0;
```

Comment:
So, FEI deposits add 1000 FEI to PCV and 1000 FEI to protocol-owned FEI, but we'll use an oracle price of 0 for FEI, so it will add 0$ to the PCV and 1000 FEI into circulation, correct ?

Reply:
^ this is confusing and should be very clearly spelled out in docs if we go forward with it

Reply:
@eswak 's comment was the intended behavior. Alternatively we can return 0 for the balance on Fei deposits to be explicit. @ditchfieldcaleb is the intended outcome confusing or just the implementation?